snap/snapcraft.yaml

```diff
@@ -65,19 +73,18 @@ parts:
       snapcraftctl build
     plugin: cmake
     configflags: ["-DRUN_IN_PLACE=FALSE", "-DCMAKE_BUILD_TYPE=Release"]
+    after:
+      - irrlicht
     build-packages:
       - cmake
       - gcc
       - g++
       - gettext
-      - imagemagick
-      - libbz2-dev
       - libcurl4-gnutls-dev
       - libfreetype6-dev
       - libglu1-mesa-dev
       - libirrlicht-dev
       - libjpeg-dev
-      - libjsoncpp-dev
```

Comment:
I'd rather use the distro package if possible. The version from minetest isn't a fork, so we're not relying on specific functionality they are adding here. By using the distro package, we gain/retain security coverage from Ubuntu devs.